baselines/dom.generated.d.ts

```diff
@@ -12658,6 +12660,177 @@ declare var HTMLPictureElement: {
+    hash: string | Algorithm;
+}
+
+interface RsaKeyAlgorithm extends KeyAlgorithm {
+    modulusLength: number;
+    publicExponent: Uint8Array;
+}
+
+interface RsaHashedKeyAlgorithm extends RsaKeyAlgorithm {
+    hash: string | Algorithm;
+}
+
+interface RsaHashedImportParams {
+    hash: string | Algorithm;
+}
+
+interface RsaPssParams {
+    saltLength: number;
+}
+
+interface RsaOaepParams extends Algorithm {
+    label?: ArrayBuffer | ArrayBufferView;
+}
+
+interface EcdsaParams extends Algorithm {
+    hash: string | Algorithm;
+}
+
+interface EcKeyGenParams extends Algorithm {
+    typedCurve: string;
+}
+
+interface EcKeyAlgorithm extends KeyAlgorithm {
+    typedCurve: string;
+}
+
+interface EcKeyImportParams {
+    typedCurve: string;
+}
+
+interface EcdhKeyDeriveParams extends Algorithm {
+    public: CryptoKey;
+}
+
+interface AesCtrParams extends Algorithm {
+    counter: ArrayBuffer | ArrayBufferView;
+    length: number;
+}
+
+interface AesKeyAlgorithm extends KeyAlgorithm {
+    length: number;
+}
+
+interface AesKeyGenParams extends Algorithm {
+    length: number;
+}
+
+interface AesDerivedKeyParams extends Algorithm {
+    length: number;
+}
+
+interface AesCbcParams extends Algorithm {
+    iv: ArrayBuffer | ArrayBufferView;
+}
+
+interface AesCmacParams extends Algorithm {
+    length: number;
+}
+
+interface AesGcmParams extends Algorithm {
+    iv: ArrayBuffer | ArrayBufferView;
+    additionalData?: ArrayBuffer | ArrayBufferView;
+    tagLength?: number;
+}
+
+interface AesCfbParams extends Algorithm {
+    iv: ArrayBuffer | ArrayBufferView;
+}
+
+interface HmacImportParams extends Algorithm {
+    hash?: string | Algorithm;
+    length?: number;
+}
+
+interface HmacKeyAlgorithm extends KeyAlgorithm {
+    hash: string | Algorithm;
+    length: number;
+}
+
+interface HmacKeyGenParams extends Algorithm {
+    hash: string | Algorithm;
+    length?: number;
+}
+
+interface DhKeyGenParams extends Algorithm {
+    prime: Uint8Array;
+    generator: Uint8Array;
+}
+
+interface DhKeyAlgorithm extends KeyAlgorithm {
+    prime: Uint8Array;
+    generator: Uint8Array;
+}
+
+interface DhKeyDeriveParams extends Algorithm {
+    public: CryptoKey;
+}
+
+interface DhImportKeyParams extends Algorithm {
+    prime: Uint8Array;
+    generator: Uint8Array;
+}
+
+interface ConcatParams extends Algorithm {
+    hash?: string | Algorithm;
+    algorithmId: Uint8Array;
+    partyUInfo: Uint8Array;
+    partyVInfo: Uint8Array;
+    publicInfo?: Uint8Array;
+    privateInfo?: Uint8Array;
+}
+
+interface HkdfCtrParams extends Algorithm {
+    hash: string | Algorithm;
+    label: ArrayBuffer | ArrayBufferView;
+    context: ArrayBuffer | ArrayBufferView;
+}
+
+interface Pbkdf2Params extends Algorithm {
+    salt: ArrayBuffer | ArrayBufferView;
+    iterations: number;
+    hash: string | Algorithm;
+}
+
+interface RsaOtherPrimesInfo {
+    r: string;
+    d: string;
+    t: string;
+}
+
+interface JsonWebKey {
+    kty: string;
```

Comment:
@zhengbli: `kty` is required because of the comment in [RFC 7517 section 4.1](https://tools.ietf.org/html/rfc7517#section-4.1) that says:

> This member MUST be present in a JWK.